CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/SallingGroup-AI-and-ML/venv-cli/tree/develop)
 
+## Minor changes
+* `venv sync` command marked as deprecated with removal planned for `v2.0`. Use `venv install <requirments>.lock` instead. [#14](https://github.com/SallingGroup-AI-and-ML/venv-cli/pull/14)
```

Comment:
```suggestion
* `venv sync` command marked as deprecated with removal planned for `v2.0`. Use `venv install <requirements>.lock` instead. [#14](https://github.com/SallingGroup-AI-and-ML/venv-cli/pull/14)
```

Reply:
Fixed in https://github.com/SallingGroup-AI-and-ML/venv-cli/commit/8665832dbee2f165127bf79150ebe1b5362312b8